content/en/docs/components/training/prometheus.md

```diff
@@ -0,0 +1,48 @@
++++
+title = "Prometheus Monitoring"
+description = "Prometheus Metrics for the Training Operator"
+weight = 70
++++
+
+This guide explains how to monitor Kubeflow training jobs using Prometheus metrics. The Training Operator exposes these metrics, providing essential insights into the status of distributed machine learning workloads.
+
+### Prometheus Metrics for Training Operator
+The Training Operator includes a built-in `/metrics` endpoint exposes Prometheus metrics. This feature is enabled by default and requires no additional configuration for basic use.
+
+#### Accessing the Metrics
+By default, the metrics are exposed on port 8080. The method to access these metrics may vary depending on your Kubernetes setup and environment.
```

Comment:
Please explain how to change the default port for metrics exporting: https://github.com/kubeflow/training-operator/blob/master/cmd/training-operator.v1/main.go#L83

Reply:
I've already added the information on changing the default port for metrics exporting. Thanks.